origin-dapp/src/components/conversation.js

```diff
@@ -233,78 +275,81 @@ class Conversation extends Component {
     }
   }
 
+  formatOfferMessage(info) {
+    const { listing = {} } = this.state
+    const { users, intl, smallScreenOrDevice, withListingSummary } = this.props
+
+    if (smallScreenOrDevice || !withListingSummary) return
+
+    const { returnValues = {}, event, timestamp } = info
+    const partyAddress = formattedAddress(returnValues.party)
+    const user = users.find((user) => formattedAddress(user.address) === partyAddress)
+    const userName = abbreviateName(user)
+    const party = userName || truncateAddress(returnValues.party)
+
+    const offerMessages = {
+      'OfferCreated': `${party} ${intl.formatMessage(
+        this.intlMessages.offerCreated
+      )}`,
+      'OfferWithdrawn': `${party} ${intl.formatMessage(
+        this.intlMessages.offerWithdrawn
+      )}`,
+      'OfferAccepted': `${party} ${intl.formatMessage(
+        this.intlMessages.offerAccepted
+      )}`,
+      'OfferDisputed': `${party} ${intl.formatMessage(
+        this.intlMessages.offerDisputed
+      )}`,
+      'OfferRuling': `${party} ${intl.formatMessage(
+        this.intlMessages.offerRuling
+      )}`,
+      'OfferFinalized': `${party} ${intl.formatMessage(
+        this.intlMessages.offerFinalized
+      )}`,
+      'OfferData': `${party} ${intl.formatMessage(
+        this.intlMessages.offerData
+      )}`,
+    }
+
+    return (
+      <div key={new Date() + Math.random()} className="purchase-info">
+        {offerMessages[event]} {listing.name} on {formatDate(timestamp)}
```

Comment:
@Natasha08 while you're here, we should probably go ahead and linkify this message to point to purchase-detail. I'm not too concerned about the styling.